lib/components/ClinicUserSelect.js

```diff
@@ -0,0 +1,162 @@
+/*
+* == BSD2 LICENSE ==
+* Copyright (c) 2014, Tidepool Project
+*
+* This program is free software; you can redistribute it and/or modify it under
+* the terms of the associated License, which is identical to the BSD 2-Clause
+* License as published by the Open Source Initiative at opensource.org.
+*
+* This program is distributed in the hope that it will be useful, but WITHOUT
+* ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+* FOR A PARTICULAR PURPOSE. See the License for more details.
+*
+* You should have received a copy of the License along with this program; if
+* not, you can obtain one from Tidepool Project at tidepool.org.
+* == BSD2 LICENSE ==
+*/
+
+var _ = require('lodash');
+var React = require('react');
+var Select = require('react-select');
+var sundial = require('sundial');
+var cx = require('classnames');
+
+var styles = require('../../styles/components/ClinicUserSelect.module.less');
+ 
+var ClinicUserSelect = React.createClass({
+  propTypes: {
+    allUsers: React.PropTypes.object.isRequired,
+    onUserChange: React.PropTypes.func.isRequired,
+    targetId: React.PropTypes.string,
+    targetUsersForUpload: React.PropTypes.array.isRequired,
+    onAddUserClick: React.PropTypes.func.isRequired,
+    setTargetUser: React.PropTypes.func.isRequired
+  },
+
+  handleChange: function(e) {
+    this.props.setTargetUser(e);
```

Comment:
Can't we just pass `this.props.setTargetUser` directly to `<Select />` as the `onChange`? Compare with `<UserDropdown />`...

Got on the track of looking at this because I found it pretty confusing at first, especially with `e` as the argument. I was like, "we're passing a _raw_ event to one of our action creators??" So at first I was gonna suggest just not using `e` when it's not a raw event, then I realized I think the whole wrapper function is unnecessary...